Docs/Configuration.tex

```diff
@@ -6054,31 +6054,31 @@ \subsection{Quirks Properties}\label{uefiquirkprops}
   \textbf{Description}: Adds delay in microseconds after \texttt{EXIT\_BOOT\_SERVICES}
   event.
 
-  This is a very ugly quirk to circumvent "Still waiting for root device" message
-  on select APTIO IV firmwares, namely ASUS Z87-Pro, when using FileVault 2 in particular.
-  It seems that for some reason they execute code in parallel to \texttt{EXIT\_BOOT\_SERVICES},
-  which results in SATA controller being inaccessible from macOS. A better approach should be
-  found in some future. Expect 3-5 seconds to be enough in case the quirk is needed.
+  This is a very rough workaround to circumvent the "Still waiting for root device" message
```

Comment:
Really need to fix that ``Still waiting for root device''.

Reply:
Presume you mean to change to \texttt{Still\ waiting\ for\ root\ device}

Reply:
Well, could be both. TeX smart quotes are inserted with ``sequences''.